pennylane/devices/tests/test_compare_default_qubit.py

```diff
@@ -68,6 +68,40 @@ def circuit(theta, phi):
         assert np.allclose(qnode(theta, phi), qnode_def(theta, phi), atol=tol(dev.shots))
         assert np.allclose(grad(theta, phi), grad_def(theta, phi), atol=tol(dev.shots))
 
+    def test_projector_expectation(self, device, tol):
```

Comment:
great idea to add it to the shared device test suite. This way, it will automatically be tested on all plugin devices.